src/vinyldns/zone.py

```diff
@@ -94,7 +95,8 @@ def from_dict(d):
             connection=map_option(d.get('connection'), ZoneConnection.from_dict),
             transfer_connection=map_option(d.get('transferConnection'), ZoneConnection.from_dict),
             acl=map_option(d.get('acl'), ZoneACL.from_dict),
-            latest_sync=map_option(d.get('latestSync'), parse_datetime)
+            latest_sync=map_option(d.get('latestSync'), parse_datetime),
+            is_test=d['isTest']
```

Comment:
is_test is a new field, so not provided by the old docker image. to address this, let's just default it to false (so pull this only if it exists, if not, set to false)